languages/python/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
@ErikSchierboom this empty blank line and the jump straight to heading level 2 (ie `##`) is in the c-sharp file is well ... I'm a bit unclear about where and how these files are rendered, but _should_ it have leading whitespace, and _should_ these files all start at a certain header level?

Reply:
The leading white space is not required, and just a mistake on my part. The header level within files should be `##`. This gives us the option to later add a `#` header on the website or in documentation if we so desire.

Reply:
Is there a centralized doc for formatting/whitespace? It would be really handy to add this as a reference, and prevent a lot of back-and-forth being spent on formatting.

Reply:
I'm removing the leading space based on the "mistake on my part" comment, then.

Reply:
I don't think we have a central formatting doc yet.